betty/service.py

```diff
@@ -0,0 +1,127 @@
+"""
+Provide a service container.
```

Comment:
"service management", perhaps? Something more helpful. Perhaps an explanation of what services are.

Reply:
Mention resource management.